templates/carbonad-standard.html.twig

```diff
@@ -0,0 +1,3 @@
+<div class="carbonad-standard">
+    <script async type="text/javascript" src="//cdn.carbonads.com/carbon.js?serve=CK7D553W&placement=wwwdoctrine-projectorg" id="_carbonads_js"></script>
```

Comment:
Is it possible to use `integrity` for carbon ads scripts?

Reply:
I will find out how often they change the javascript.

Reply:
We could use it, but every time they change the javascript, it would break.

Reply:
I don't find anything about Carbonads script versions in a quick Google search.

Reply:
I talked to carbon folks and they said the js doesn't change that often and that we can use the integrity attribute so I will add it.

Reply:
Ok more info. The script is updated every 1 to 2 months and they don't recommend using sri hashes.